src/services/__tests__/messagesService.test.ts

```diff
@@ -263,6 +281,23 @@ describe("MessageService#getServicesByRecpient", () => {
     });
   });
 
+  it("returns an error if the getServicesByRecpient API returns an 429 HTTP error from the upstream API", async () => {
+    mockGetServicesByRecipient.mockImplementation(() =>
+      t.success(tooManyReqApiMessagesResponse)
+    );
+
+    const service = new MessageService(api);
+
+    expect.assertions(2);
+    return service.getServicesByRecipient(mockedUser).then(e => {
```

Comment:
async tests don't need to return the promise (see the other tests as an example), you must use `await` here

Reply:
Fix in e0e50dd